public/js/ajax-airline-handler.js

```diff
@@ -0,0 +1,151 @@
+$('form').submit(function(event){
+    event.preventDefault();
+})
+
+const createAirline = () => {
+    fetch('api/airlines', {
+        method: 'POST',
+        headers: {
+            'Content-type': 'application/x-www-form-urlencoded',
+            'X-Requested-With': 'XMLHttpRequest',
+            'X-CSRF-TOKEN': $('meta[name="csrf-token"]').attr('content')
+        },
+        body: new URLSearchParams($('#add-airline-form').serialize())
+    })
+
+    .then(response => {
+        if (response.ok) {
+            $('#name-error').text('');
+            $('#description-error').text('');
+            document.forms["add-airline-form"].reset();
+
+            return response.json();
+
+        } else {
```

Comment:
When an `if` statement has a `return` statement, you don't need an `else`. Especially here, where only two things can happen (either the `if` evaluates to true and you return the response as JSON, or you show the validation errors).